buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1815,6 +1815,58 @@ class BeamModulePlugin implements Plugin<Project> {
           }
         }
       }
+
+      project.ext.addPortableWordCountTask = { String nameSuffix, boolean isStreaming ->
```

Comment:
We can use implicit argument to get nameSuffix and isStreaming as we do in https://github.com/apache/beam/blob/master/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L1649

Reply:
Good to know, thanks. I find explicit way more reader-friendly. Unless there is a reason (long list of optional parameters, etc), I think explicit is better than implicit.

Reply:
I have similar feeling but I think in groovy land, implicit is more popular because we don't have to change method signature to add new arguments or remove arguments.

Reply:
PTAL. I couldn't find such recommendation in Groovy style guide http://groovy-lang.org/style-guide.html. I moved the streaming parameter into an internal helper function, and it only has 1 parameter right now. Making it implicit requires adding another "Configuration" class, which at this point seems unnecessary boilerplate. I agree that passing maps is better when we have large number of parameters, similar how service requests/responses are JSON / ProtoBufs, but I don't think we should follow this everywhere.